### DISCUSSION TITLE
My account is hidden, I’d like it not to be.

### DISCUSSION THREAD

**Discussion by Zerflin** [mastodon/mastodon#23296] [Q&A]
My account, @benjancewicz@mastodon.social 

I have emailed staff@mastodon.social several times, but with no reply.

I am indeed a real person, and would be glad to verify this in any way I need to.

 ![image](https://user-images.githubusercontent.com/13923058/215338845-4cbeb3be-5031-42e0-8ff2-f21dfdcbbf08.jpeg)

**Comment by nightpool** [comment]
This forum is for the development and maintenance of the Mastodon server software. This is a free software package that can be run by anyone who wants to provide their own, connected, federated social media software. It is not a website, just code that others use to run a website.

The people who maintain this software are, for the most part, not involved or associated with the individual websites that people run or sign up on. You'll have to contact the person running your server to get more details about what may or may not be happening here.
